tests/jvm/src/test/scala-2.12/scala/meta/tests/semanticdb/MemberSuite.scala

```diff
@@ -160,4 +160,42 @@ class MemberSuite extends DatabaseSuite(SemanticdbMode.Slim, MemberMode.All) {
     """.stripMargin
   )
 
+  members(
+    """
+      |object a {
+      |  val run = new java.lang.Runnable {
+      |    def run(): Unit = ()
+      |  }
```

Comment:
Can we add a test case to check what happens for
```scala
class Bar()
trait Foo { def bar: Int }
```

Reply:
It's empty, as expected?

```scala
members(
    """
      |object a {
      |  class Bar()  
      |  trait Foo { def bar: Int }  
      |}""".stripMargin,
    ""
  )
```

Reply:
Good, as expected!